.github/workflows/main.yml

```diff
@@ -17,6 +17,7 @@ jobs:
         uses:
           malikabhi05/test-ac/runCI@master
         with:
+          PROJECT_NAME: ${{ github.event.repository.name }}
```

Comment:
Nice, and this is a required arg in the composite action (or whatever its called)?

Reply:
Yes, this is required. It is used by the ci.sh script for the docker path